servers/cromwell/README.md

```diff
@@ -264,6 +264,78 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
   - The `forcedLogoutDomains` setting is an array of user domains where this should apply.
   - The `forcedLogoutTime` is the amount of inactive time (in milliseconds) that will trigger an automatic sign-out.
 
+
+- (Optional, CromIAM and SAM) Configure fields to display
+  - **Note:** If you want to use use Job Manager against CromIAM and you also have access to a SAM server to handle authentication, the `capabilities_config.json` can be set up with the `outsideAuth` setting in the `authentication` section, which will allow job manager to get Google Pipelines operation details and the contents of log files stored in the job's execution directory:
```

Comment:
"...If you want to **use use** Job Manager..."

Reply:
it was actually in 3 places, so thanks for the catch!  fixed